Cargo.toml

```diff
@@ -18,12 +18,17 @@ proc-macro = true
 syn = "1.0"
 quote = "1.0"
 proc-macro2 = "1.0"
-edn-rs = "0.15.0"
+edn-rs = "0.16.1"
 
 [dev-dependencies]
 trybuild = { version = "1.0", features = ["diff"] }
+criterion = "0.3"
 
 [dev-dependencies.cargo-husky]
 version = "1"
 default-features = false
 features = ["user-hooks"]
+
+[[bench]]
+name = "deser"
```

Comment:
I don't see why the bench can't be `deserialize`. What do you think?

Reply:
Since the test and implementation are the full word.

Reply:
just lazyness